Zukini/Hooks.cs

```diff
@@ -55,8 +55,7 @@ protected FeatureContext FeatureContext
         protected void BeforeScenario()
         {
             // Create a property bucket so we have a place to store values between steps
-            var propertyBucket = new PropertyBucket();
-            _objectContainer.RegisterInstanceAs<PropertyBucket>(propertyBucket);
+            var propertyBucket = _objectContainer.Resolve<PropertyBucket>();
```

Comment:
Is this the only change you made? Is the PropertyBucket being added to the ObjectContainer somewhere else?